plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -306,5 +306,5 @@ def _move_file(self, unit):
         :param unit: a pulp unit
         :type  unit: pulp.plugins.model.Unit
         """
-        _logger.debug('moving files in to place for Unit {}'.format(unit))
+        _logger.debug('moving files in to place for Unit %s' % unit)
```

Comment:
I agree with @seandst: let's change this line to:

```
_logger.debug('moving files in to place for Unit {unit}'.format(unit=unit))
```

Let me know when you've made this change and I'll merge.